filters/imagefilter.go

```diff
@@ -101,3 +103,10 @@ func transformImage(out *io.PipeWriter, image *bimg.Image, opts *bimg.Options) e
 
 	return err
 }
+
+func applyDefaults(o *bimg.Options) *bimg.Options {
+	if o.Quality == 0 {
```

Comment:
The problem is [here](https://github.com/h2non/bimg/blob/master/resize.go#L140):
every time the Process is called, if you don't set the quality, the quality is set to 80.
In the case of skrop, if you put more than one filter in one configuration of the eskip file, for example you have a configuration that crop, resize and convert the image type, EACH filter lower the quality.
The more filter you concatenate the lower the resulting image quality